doc/vm-tools/qvm-copy.rst

```diff
@@ -1,6 +1,25 @@
+==============
+qvm-copy
+==============
+
+NAME
+====
+qvm-copy - copy specified files to specified destination VM
+
+SYNOPSIS
+========
+| qvm-copy [--without-progress] dest_vmname file [file]+
```

Comment:
No `dest_vmname` argument in this tool (that's actually its only difference from qvm-copy-to-vm).